.github/workflows/build.yaml

```diff
@@ -48,7 +48,36 @@ jobs:
           cmake3 -DCMT_TESTS=on -DCMT_DEV=on .
           make
 
-  build-unix:
+  build-unix-arm64:
+    name: Build sources on arm64 for ${{ matrix.os }} - ${{ matrix.compiler }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 48
```

Comment:
Seems excessive, how come?

Reply:
it was a copy + paste, but we'll leave it as the default.